client/oci_env/utils.py

```diff
@@ -71,6 +71,10 @@ def get_config(env_file):
         "API_PROTOCOL": "http",
         "API_HOST": "localhost",
         "API_PORT": "5001",
+
+        # A port dedicated for exposing generated docs
+        "DOCS_PORT": "12345",
+        "NGINX_DOCS_PORT": user_preferences.get("DOCS_PORT", "12345"),
```

Comment:
This is trying to address the comment from https://github.com/pulp/oci_env/pull/102#discussion_r1151032579.

Reply:
With this the ports will not collide when running two oci-envs?